README.md

```diff
@@ -1,48 +1,84 @@
+  - [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-This repository includes files with plain SQL that can be used to recreate a database:
+# 📖 [Vet Clinic Database] <a name="about-project"></a>
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+**Vet Clinic Database** is a database to manage data stored for a vet clinic. It uses PostgreSQL
 
+## 🛠 Built With <a name="built-with"></a>
 
-## Authors
+### Tech Stack <a name="tech-stack"></a>
 
-👤 **Author1**
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.pgadmin.org/">pgAdmin</a></li>
+  </ul>
+</details>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### Key Features <a name="key-features"></a>
 
-👤 **Author2**
+- **DB name - vet_clinic**
+- **Tables - animals**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [No Live Demo]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kwasi Antwi Baayeh**
+
+- GitHub: [@githubhandle](https://github.com/Baayeh)
+- Twitter: [@twitterhandle](https://twitter.com/Cest_Baayeh)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/kabaayeh)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank
 
-Give a ⭐️ if you like this project!
+- [Microverse](https://www.microverse.org/)
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [OPTIONAL] Your README is well documented 💯 I just noticed that the MIT/LICENSE link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
I have included the MIT.md file. Thank you